cset-workflow/includes/metplus_grid_stat.cylc

```diff
@@ -0,0 +1,16 @@
+{% if METPLUS_GRID_STAT %}
+[scheduling]
+    [[graph]]
+    {{CSET_CYCLE_PERIOD}} = """
+    metplus_grid_stat => housekeeping_full
```

Comment:
I think it should depend on the data fetching task. But it will involve other METPlus apps too, I was thinking that we need an issue on incorporating fully the METPlus tasks into the cset workflow. This also includes using the PARALLEL/COLLATE construct.